src/examples/mfem_common_plane.cpp

```diff
@@ -0,0 +1,429 @@
+// Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+// other Tribol Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (MIT)
+
+#include <set>
+
+#ifdef TRIBOL_USE_UMPIRE
+// Umpire includes
+#include "umpire/ResourceManager.hpp"
+#endif
+
+// MFEM includes
+#include "mfem.hpp"
+
+// Axom includes
+#include "axom/CLI11.hpp"
+#include "axom/slic.hpp"
+
+// Redecomp includes
+#include "redecomp/redecomp.hpp"
+
+// Tribol includes
+#include "tribol/common/Parameters.hpp"
+#include "tribol/config.hpp"
+#include "tribol/interface/tribol.hpp"
+
+/// Simple central difference method.
+class CentralDiffSolver : public mfem::SecondOrderODESolver
```

Comment:
Can we pull this class and the explicit mechanics out into a examples header file? Maybe not on example_common.hpp, but maybe introduce a new file (solids_update.hpp or something that nods at both explicit solids update and time integration methods)?

Reply:
Done, added to `TestUtils.*pp` under the `mfem_ext` namespace.